receiver/udp.go

```diff
@@ -56,6 +63,9 @@ func (ud *UDP) process() {
 // listening for incoming UDP messages on the configured address. Start
 // never returns.
 func (ud *UDP) Start() error {
+	if ud.PacketSize < 0 || ud.PacketSize > UDP_MAX_READ_SIZE {
+		return errors.New("invalid udp packet size, maximum udp read size is between 0 and " + strconv.Itoa(UDP_MAX_READ_SIZE))
+	}
```

Comment:
This check could be done as part of `Verify()`. I see that this receiver doesn't have it yet, but if you add it there it makes more sense, and it'll stop the program from starting up in the first place instead of erroring upon input.

`func (ud *UDP) Verify() error {}` and then return an error from there if the configuration is wrong. (Look at receivers/http.go for an example of the `Verify()` func)

You should also return the `skogul.Error` struct instead of using `errors`, putting the error message in `Reason` and write `udp-receiver` in `Source`.